Junk4Dollars-app/Auctions/AuctionListViewController.swift

```diff
@@ -34,20 +36,17 @@ public class AuctionListViewController: UIViewController {
         }
     }
 
-    func loadAuctions() -> Void {
-        self.apiServices.getAllAuctions { result in
-            switch result {
-                case .success(let auctions):
-                    self.auctionsDataSource.setAuctions(auctions)
-                    DispatchQueue.main.async {
-                       self.auctionTableView.reloadData()
-                    }
-                case .error(let message):
-                    print("An error occurred: \(message)")
-            }
+    public func refreshAuctions(_ auctions: [Auction]) -> Void {
+        self.auctionsDataSource.setAuctions(auctions)
+        DispatchQueue.main.async {
+            self.auctionTableView.reloadData()
         }
     }
 
+    func loadAuctions() -> Void {
```

Comment:
Presuming that this function is only called once in the production code and that this function is now a single line, it can probably be inlined into where it is called.